mammoth/opts.py

```diff
@@ -326,6 +326,12 @@ def model_opts(parser):
         "and decoder. Need to use shared dictionary for this "
         "option.",
     )
+    group.add(
+        '--enable_embeddingless',
+        '-enable_embeddingless',
+        action='store_true',
+        help="Enable the use of embeddingless models.",
```

Comment:
Perhaps add a reference here or a tiny description of what embeddingless is here -- e.g. byte level something something implementaiton based on citation_name et al. (2045)